include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -16,6 +16,8 @@
 #ifndef _ONEDPL_CONFIG_H
 #define _ONEDPL_CONFIG_H
 
+#include "oneapi/dpl/internal/common_config.h"
```

Comment:
I believe we had a agreement that common_config should be included in (all) public headers only

Reply:
@akukanov what do you think about this?

Reply:
Fixed

Reply:
This change was made by me and approved in some previous PR https://github.com/oneapi-src/oneDPL/pull/642

Reply:
I guess I do not understand the response. In #642, this header file was included into pstl/experimental/algorithm which is, I think, a public header file.

Is it technically needed to add `common_config.h` to `onedpl_config.h` (i.e. otherwise there would be an error), or is it only included to indicate where the C++ version macro comes from?

Reply:
> Member
"I believe we had a agreement that common_config should be included in (all) public headers only" - yes, it's true.